src/RequestHandler/outofprocess/serverprocess.cxx

```diff
@@ -528,15 +528,20 @@ SERVER_PROCESS::PostStartCheck(
             // make sure the process is still running
             if (processStatus != STILL_ACTIVE)
             {
-                hr = E_APPLICATION_ACTIVATION_EXEC_FAILURE;
-                strEventMsg.SafeSnwprintf(
-                    ASPNETCORE_EVENT_PROCESS_START_STATUS_ERROR_MSG,
-                    m_struAppFullPath.QueryStr(),
-                    m_struPhysicalPath.QueryStr(),
-                    m_struCommandLine.QueryStr(),
-                    hr,
-                    processStatus);
-                goto Finished;
+                // double check
```

Comment:
Why? Do we expect it to change in the millisecond since last check?

Reply:
as I stated, sometime the call returned 0, while the process is still alive per the log. Double check to avoid false alert. This issue was found in VS team test.

Reply:
Do we know that second would return correct value?

Reply:
We don't know. We can only do double check. With those change, VS tests seems stable now.